UIAutomationTests/TestsAuthenticationWindowsKerberos.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Xunit;
+
+namespace UIAutomationTests
+{
+    [Collection("VisualStudioUIAutomationTestCollection")]
+    public class TestsAuthenticationWindowsKerberos
+    {
+        UIAutomationTestFixture testFixture;
+        public TestsAuthenticationWindowsKerberos(UIAutomationTestFixture tf)
+        {
+            testFixture = tf;
+        }
+        //This is a unit test where we'll check if the OdbcConnection record that gets passed on to the Odbc Query or Datasource function is correct.
+        [Fact]
+        public void TestWindowsKerberosAuthenticationConnectionStringRecord()
+        {
+            var (sbPqFileStr, dnsRecord) = CreateKerberosConnectionStringPqTestFile();
+
+            var (error, grid) = testFixture.Test(sbPqFileStr, UIAutomationTestFixture.AuthenticationMethod.None);
+
+            Assert.True(String.IsNullOrWhiteSpace(error), $@"Errormessage: {error}");
+
+            string connectionStrKerberosServicenameField = "KERBEROSSERVICENAME";
+            //assert there's a row with a Kerberos service record field
+            int? kerberosServiceNameRecordRow = Utilities.FindRow(grid, 0, connectionStrKerberosServicenameField);
+            Assert.True(kerberosServiceNameRecordRow.HasValue, "KERBEROSSERVICENAME is missing from the Connection String Record.");
+            Assert.True(grid.Rows[kerberosServiceNameRecordRow.Value].Cells[1].Value == "exasol/" + dnsRecord);
```

Comment:
maybe we can extract `grid.Rows[kerberosServiceNameRecordRow.Value].Cells[1].Value` to a `getConnectionString()` method or something similar

Reply:
Will do if more work on it is performed in the future.